ql/src/java/org/apache/hadoop/hive/ql/ddl/DDLUtils.java

```diff
@@ -217,4 +218,11 @@ public static void setColumnsAndStorePartitionTransformSpecOfTable(
       partCols.ifPresent(tbl::setPartCols);
     }
   }
+
+  public static void validateTableIsIceberg(org.apache.hadoop.hive.ql.metadata.Table table)
```

Comment:
doesn't it duplicate `HiveTableOperations.validateTableIsIceberg`, could we keep just one version of it 
````
  static void validateTableIsIceberg(Table table, String fullName) {
    String tableType = table.getParameters().get(TABLE_TYPE_PROP);
    NoSuchIcebergTableException.check(tableType != null && tableType.equalsIgnoreCase(ICEBERG_TABLE_TYPE_VALUE),
        "Not an iceberg table: %s (type=%s)", fullName, tableType);
  }
````

Reply:
yep, it duplicates `HiveTableOperations.validateTableIsIceberg` as DDLTask in`ql `  package can not import iceberg class.
 
I changed the thrown exception to` Not an iceberg table: %s (type=%s)` , this means the same thing as `HiveTableOperations.validateTableIsIceberg`